### ISSUE TITLE
Is it possible to run rar2fs within Windows Subsystem for Linux?

### ISSUE THREAD

**Issue by psisis** [hasse69/rar2fs#141]
First of all I appreciate this program, I have not found anything quite like it. Equivalents on Windows like WinArchiver are useless, since they “mount” by extracting to AppData, which I could do myself.

I tried installing rar2fs on Arch Linux within WSL2. It worked in itself, and it successfully mounts an archive to home dir. I also tried playing a video archive with mpv and it works fine and very fast. The issue is whether I can access this mounted archive from Windows Explorer and Windows-based programs.

The first thing I tried was mounting an archive to my Windows drive, both Ntfs and exFat. But fuse fails and it says mounting over such filesystems is not allowed. The Linux drive is a virtual .vhdx file, and through Windows you reach it as a network drive, apparently via 9P protocol. If I mount an archive and try to open the folder through Explorer it does not work, it just doesn’t open. And finding it through cmd, says file not found.

I understand if I can’t get help for such a niche application, but if anyone knows that would be great.

**Comment by hasse69** [comment]
The method(s) I am aware of and use myself is CIFS/Samba or NFS shares from Linux to Windows.
If you wish to run rar2fs locally on your Windows host you can compile rar2fs using Cygwin and use WinFSP. See the wiki for instructions on how to do that.

**Comment by psisis** [comment]
I finally got it to work. NFS would not find the mounted file. With samba, as soon as you mounted it the folder disappeared, or it would tell you there is no access. I tried chmod and chown, but to no effect. I then found the command `rar2fs -o allow_other`. Now it works. Nfs still doesn't find it, native 9P does not, only samba will. But I'll take it.

Might be good if there is a wiki entry for it?

I tried to compile rar2fs with msys some months ago, both dokan and winfsp, but it wouldn't. I know cygwin probably works, but I have no other use for cygwin.
